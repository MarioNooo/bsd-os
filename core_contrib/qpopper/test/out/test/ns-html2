Return-Path: <lgl@booray.new-era.com>
Received: from nala.qualcomm.com (nala.qualcomm.com [129.46.50.44])
	by booray.new-era.com (8.8.5/8.8.5) with ESMTP id TAA24777
	for <lgl@island-resort.com>; Thu, 2 Oct 1997 19:58:26 -0400
Received: from llundbla (llundblade.qualcomm.com [129.46.137.63]) by nala.qualcomm.com (8.8.5/1.4/8.7.2/1.13) with ESMTP id QAA21356; Thu, 2 Oct 1997 16:57:54 -0700 (PDT)
Message-ID: <34343205.F6BC57E3@booray.new-era.com>
Date: Thu, 02 Oct 1997 16:45:10 -0700
From: Laurence Lundblade <lgl@booray.new-era.com>
X-Mailer: Mozilla 4.01 [en] (WinNT; I)
MIME-Version: 1.0
To: lgl@island-resort.com, lgl@qualcomm.com
Subject: test
X-Priority: 3 (Normal)
Content-Type: text/html; charset=us-ascii
Content-Transfer-Encoding: 7bit

<HTML>
<U>This is HTML only</U>
<BR>&nbsp;</HTML>
