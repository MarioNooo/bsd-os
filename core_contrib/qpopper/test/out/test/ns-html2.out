Date: Thu, 02 Oct 1997 16:45:10 -0700
From: Laurence Lundblade <lgl@booray.new-era.com>
To: lgl@island-resort.com, lgl@qualcomm.com
Subject: test

This is HTML only 
  